input/ignoreWarnings.txt

```diff
@@ -1,6 +1,10 @@
 == Suppressed Messages ==
 
-# Add warning and/or information messages here after you've confirmed that they aren't really a problem
-# instructions for ignoreWarnings.txt https://confluence.hl7.org/pages/viewpage.action?pageId=66938614#ImplementationGuideParameters-ManagingWarningsandHints
-# (And include comments like this justifying why)
```

Comment:
Need to keep these instructional messages, this is a template IG that is supposed to provide documentation on how to build a content IG, this is an important part of that